marker_api/src/ast/expr.rs

```diff
@@ -61,7 +61,7 @@ pub enum ExprKind<'ast> {
     UnaryOp(&'ast UnaryOpExpr<'ast>),
     Ref(&'ast RefExpr<'ast>),
     BinaryOp(&'ast BinaryOpExpr<'ast>),
-    QuestionMark(&'ast QuestionMarkExpr<'ast>),
+    QuestionMark(&'ast TryExpr<'ast>),
```

Comment:
I suppose the variant needs to be renamed as well. But.. how should it be called? Just `Try`? That may be confusing with the try blocks. I would be fine with this renaming if it wasn't for the try blocks. It may be very confusing when try blocks become stable to have a TryExpr, which isn't a try block

Reply:
Good catch! For variant name, I would currently call it `Try`. When the `TryBlock` is added, I would first look, if it can be merged with the `BlockExpr` and if not, add a new expression node, called `TryExpr`, here. That's also how syn, calls these nodes: https://docs.rs/syn/latest/syn/enum.Expr.html#variant.Try

Reply:
I've added the RFC to #240

Reply:
Okay, I think it should be good to be consistent with `syn`. If `syn` calls this a `TryExpr`, I'm fine with this change